bin/dbatools-buildref-index.json

```diff
@@ -4181,7 +4181,7 @@
         {
             "Version": "15.0.2000",
             "SP": "RTM",
-            "SupportedUntil": "2025-07-01"
+            "SupportedUntil": "2030-01-08"

```

Comment:
This should not have been touched, but wasn't paying attention so I'll fix it in another PR.